connector/launchers/api/build.gradle.kts

```diff
@@ -0,0 +1,30 @@
+plugins {
+    `java-library`
+    id("application")
+    alias(libs.plugins.shadow)
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation(libs.edc.auth.spi)
+    implementation(libs.edc.web.spi)
+    implementation(libs.edc.spi.token)
+    implementation(libs.edc.data.plane.public.api)
+
+
+    implementation("com.nimbusds:nimbus-jose-jwt:9.37.3")
+    implementation("com.nimbusds:oauth2-oidc-sdk:11.3")
```

Comment:
these versions should stay in the version catalog

Reply:
Oh yes, I'll fix that